src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/MulticlassNaiveBayesTrainer.cs

```diff
@@ -222,7 +223,7 @@ internal static CommonOutputs.MulticlassClassificationOutput TrainMulticlassNaiv
     /// </summary>
     public sealed class NaiveBayesMulticlassModelParameters :
         ModelParametersBase<VBuffer<float>>,
-        IValueMapper
+        IValueMapper, ISingleCanSaveOnnx
     {
```

Comment:
Nit: The convention here seems to be add new base classes on a new line. Can you please add ISingleCanSaveOnnx on a new line?